ci/deps/travis-36-doc.yaml

```diff
@@ -21,6 +21,7 @@ dependencies:
   - notebook
   - numexpr
   - numpy=1.13*
+  - numpydoc
```

Comment:
do we have a min version required?

Reply:
I don't know exactly. Currently it uses the latest which is 0.8.0.